benchmarks/hough_transform/oop_c#/Program.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Drawing;
+using System.Linq;
+
+namespace oop_c_
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var hough = new HoughTransform("benchmarks/hough_transform/Pentagon.png", 480, 640);
+            var output = hough.ComputeTransformation();
+            System.Console.WriteLine(output.Cast<int>().Sum());

```

Comment:
Måske ikke helt oop, at køre .Sum. Kunne være en for løkke som i oop f#

Reply:
Men så længe der ikke er nogle higher order functions er det vel fint...